src/marker_cache.cpp

```diff
@@ -175,7 +175,15 @@ std::shared_ptr<mapnik::marker const> marker_cache::find(std::string const& uri,
             svg_path_adapter svg_path(stl_storage);
             svg_converter_type svg(svg_path, marker_path->attributes());
             svg_parser p(svg);
-            p.parse_from_string(known_svg_string);
+
+            if (!p.parse_from_string(known_svg_string))
+            {
+                for (auto const& msg : p.error_messages())
+                {
+                    std::cerr << "SVG PARSING ERROR:\"" << msg << "\"" << std::endl;
```

Comment:
@artemp Should these be going to `std::cerr` or `MAPNIK_LOG_ERROR(svg_parser)`? I believe this is the cause of #3007.